dot_config/rulesync/exact_dot_rulesync/skills/archon/guides/config.md

```
@@ -0,0 +1,226 @@
+Header: "Global option"
+Question: "What would you like to change in the global config?"
+Options:
+  1. "Bot name" — Display name shown in messages (current: {value}, default: Archon)
+  2. "Default assistant" — AI assistant for new repos (current: {value}, default: claude)
+  3. "Streaming modes" — How responses are delivered per platform
+  4. "Concurrency" — Max concurrent AI conversations (current: {value}, default: 10)
+```
+
+If "Streaming modes" is selected, follow up with:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)